src/value/tag.rs

```diff
@@ -1,10 +1,10 @@
 use std::fmt;
 use std::sync::atomic::Ordering;
 
-#[cfg(not(feature="fallback-atomic"))]
+#[cfg(target_pointer_width = "64")]
```

Comment:
Instead of having to do the `cfg` 4 times in the file, cut the `cfg`s by two and use a fully qualified path in the `static` decl:

```rust
#[cfg(...)]
static COUNTER: atomic::Atomic<u64> = atomic::Atomic::new(1);
```